tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md

```diff
@@ -63,7 +63,7 @@ In addition to global accounts, directories, and subaccounts, the Cloud Foundry
 ### ABAP
 Within the Cloud Foundry environment, you can create a new space for ABAP development. This is what we refer to as the ABAP environment. It allows you to create extensions for ABAP-based products, such as SAP S/4HANA Cloud, and develop new cloud applications. You can transform existing ABAP-based custom code or extensions to the cloud.
 
-The ABAP environment is based on the latest ABAP platform cloud release that is also used for SAP S/4HANA Cloud. It leverages the innovations provided by SAP HANA. The software stack contains standard technology components that are familiar from the standalone Application Server ABAP. The ABAP environment supports the ABAP RESTful Application Programming Model including SAP Fiori and Core Data Services (CDS). SAP Services and APIs are offered according to a new whitelisting approach. The ABAP environment provides you technical access to SAP Business Technology Platform services, such as destination service, integration, machine learning, and IoT.
+The ABAP environment is based on the latest ABAP platform cloud release that is also used for SAP S/4HANA Cloud. It leverages the innovations provided by SAP HANA. The software stack contains standard technology components that are familiar from the standalone Application Server ABAP. The ABAP environment supports the ABAP RESTful Application Programming Model including SAP Fiori and Core Data Services (CDS). SAP Services and APIs are offered according to a new allowlist approach. The ABAP environment provides you technical access to SAP Business Technology Platform services, such as destination service, integration, machine learning, and IoT.
```

Comment:
I think that it should be "allow list". Please check.

Reply:
No, actually it should be "allowlisting" but the spellchecker would not let me publish that.
https://help.sap.com/doc/b0322267728e48a28b0c8ee7dd1ab4c7/1.0/en-US/Inclusive%20Language%20Guidelines.pdf

Reply:
@thecodester Do you have an idea on how to solve this?